### DISCUSSION TITLE
How to remake the similar HTMLOverlay in v7?

### DISCUSSION THREAD

**Discussion by sangdth** [uber/react-map-gl#1717] [Q&A] [answered]
**Update**:
Ahh, I got it. I will close this later.

---

I tried to look at the example here https://github.com/visgl/react-map-gl/blob/7.0-release/examples/custom-overlay/src/custom-overlay.tsx but don't see anything related to the old HTMLOverlay.

Could someone give me a hint?

I just want to render an input with a list of search results on top of the map.
```
<MyDivOverlay> // Just a div
   <Input ...> // My input
   <SearchResults ... /> // my search results
</MyDivOverlay>
```
Thanks in advance.

**Comment by Pessimistress** [comment, marked as answer]
It seems that the example is still using beta API. I have opened #1718 to update it.

**Reply by sangdth** [reply]
Thank you! Now I understand it better!
